src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
Requiring a method to be final is as breaking as requiring a new (protected) method (`applyFix`).
Users are not affected, maintainers of side-projects (that require PHP-CS-Fixer) are affected.